docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -70,19 +70,44 @@ KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
-* Edit /etc/kubernetes/controller-manager to appear as such:
+* Addition of nodes.
+
+* Create following node.json file on kubernetes master node:
+
+```json
+{
+  "id": "fed-node",
+  "kind": "Minion",
+  "apiVersion": "v1beta1",
+  "resources": {
+    "capacity": {
+      "cpu": 1,
+      "memory": 1073741824
```

Comment:
Let's not include a capacity statement.  Let just the the embedded cadvisor code supply this information.